src/Rating.php

```diff
@@ -20,8 +20,12 @@ class Rating extends ClientBase
      *
```

Comment:
**CHANGE REQUEST**

Should document the options available i.e. Guzzle options + service codes + option codes.